Documentation/building/linux-instructions.md

```diff
@@ -97,122 +133,22 @@ After the build is completed, there should some files placed in `bin/Product/Lin
 * `libcoreclr.so`: The CoreCLR runtime itself.
 * `System.Private.CoreLib.dll`: Microsoft Core Library.
 
-Build the Framework
+Create the coreOverlay
 ===================
 
 ```
-ellismg@linux:~/git/corefx$ ./build.sh
+./build-test.sh generatelayoutonly
 ```
 
-After the build is complete you will be able to find the output in the `bin` folder.
+After the build is complete you will be able to find the output in the `bin/tests/Linux.x64.Debug/Tests/Core_Root` folder.
 
-Build for ARM/Linux
+Running a single test
 ===================
 
-The CI system and official builds use Docker to build ARM for Linux (for example, see the latest build [here](https://ci.dot.net/job/dotnet_coreclr/job/master/job/arm_cross_checked_ubuntu/lastSuccessfulBuild/consoleText)). The Docker container has pre-built rootfs directories containing the required tools. To build this way, do the following:
+After `build-test.sh` is run, corerun from the Core_Root folder is ready to be run. This can be done by using the full absolute path to corerun, or by setting
+an environment variable to the Core_Root folder.
```

Comment:
I think there is no need to artificially justify by inserting newlines. The browser can do it. Some people work with wider github windows using browser extensions

Reply:
But I don't think it hurts, and when you're viewing raw diffs (like I am now), it makes it a bit easier.

Reply:
It looks like our convention is to not insert new lines, so I will follow convention.